opentelemetry-aws/src/lib.rs

```diff
@@ -86,6 +86,107 @@ pub mod trace {
         _private: (),
     }
 
+    pub fn deserialize_span_context(value: &str) -> Result<SpanContext, ()> {
```

Comment:
`Result<.., ()>` is considered to be unidiomatic, just use `Option<..>` for this case instead of allowing the clippy linlt.

I would probably name this `span_context_from_str()`.

Reply:
Changed the return type, and renamed the function name